tests/storage/filesd_metadata_test.py

```diff
@@ -88,20 +88,21 @@
 _SHA_CKSUM=769baef6c65aeef08cf6d177c2c44046b2aac877
 """,
 }
+BACKENDS = userstorage.load_config("storage.py").BACKENDS
 
 Storage = collections.namedtuple("Storage", "path, block_size, alignment")
 
 
 @pytest.fixture(
     params=[
         pytest.param(
-            (userstorage.PATHS["file-512"], sc.ALIGNMENT_1M),
+            (BACKENDS["file-512"], sc.ALIGNMENT_1M),
             id="file-512-1m"),
         pytest.param(
-            (userstorage.PATHS["file-4k"], sc.ALIGNMENT_1M),
+            (BACKENDS["file-4k"], sc.ALIGNMENT_1M),
             id="file-4k-1m"),
         pytest.param(
-            (userstorage.PATHS["file-4k"], sc.ALIGNMENT_2M),
+            (BACKENDS["file-4k"], sc.ALIGNMENT_2M),
             id="file-4k-2m"),
     ],
 )
```

Comment:
I already commented - we need to call setup() and teardown() for every test. Using the parameters is not enough. We need to add a wrapper calling setup and teardown.